docs/webpack.md

```diff
@@ -155,9 +159,8 @@ fileLoader.exclude = /\.(svg)$/i
 
 ### Url Loader
 
+#### config/webpack/loaders/url.js
 ```js
-// config/webpack/loaders/url.js
```

Comment:
This might be a mistake, but I may have just never used webpacker like this.

Reply:
I don't understand this comment. The PR adds headers.